benchto-it/src/test/resources/benchmarks/test_throughput_test.yaml

```diff
@@ -0,0 +1,14 @@
+datasource: test_datasource
+query-names: ${query}.sql
```

Comment:
nit: make it a little more as pseudo throughput test? 
I mean use short query-names list, with concurrency and runs > 3?

Reply:
> I mean use short query-names list
no difference on my eye
> concurrency and runs > 4
done